tgui/packages/tgui/interfaces/common/Operating.js

```diff
@@ -0,0 +1,41 @@
+import { Flex, Icon, Dimmer } from  '../../components';
+const PropTypes = require('prop-types');
+
+/**
+ * Displays a beaker's contents
+ * @property {object} props
+ */
+export const Operating = (props) => {
+  const { operating, name } = props;
+  if (operating) {
+    return (
+      <Dimmer>
```

Comment:
Biogenerator also has `const Processing` which is the same as this, can you replace it there too? Thank you!

Reply:
Has been replaced.